terms.html

```diff
@@ -246,6 +246,15 @@
 representation metadata and a potentially unbounded stream of representation
 data." A <a>DID document</a> is a representation of information describing a
 <a>DID subject</a>. See <a href="#representations"></a>.
+  </dd>
+
+    <dt><dfn data-lt="representation-specific entry|non-representation-specific entry">representation-specific entries</dfn></dt>
+
+  <dd>
+Entries in a <a>DID document</a> whose meaning is particular to a specific
+<a>representation</a>. Defined in <a href="#data-model"></a> and
+<a href="#representations"></a>. For example, <a><code>@context</code></a> in the
+<a href="#json-ld">JSON-LD representation</a>.
```

Comment:
```suggestion
<a href="#json-ld">JSON-LD representation</a> is a <em>representation-specific entry</em>.
```

Reply:
Applied in e8ebf2d1e4f2b5e78459fc07f11ceae617d01df1